etk-asm/src/asm.rs

```diff
@@ -179,4 +282,59 @@ mod tests {
 
         Ok(())
     }
+
+    #[test]
+    fn assemble_include() -> Result<(), Error> {
+        let f = new_file!("push1 42");
+        let nodes = nodes![
+            Op::Push1(Imm::from(1)),
+            Node::Include(f.path().to_owned()),
+            Op::Push1(Imm::from(2)),
+        ];
+        let mut asm = Assembler::new();
+        let sz = asm.push_all(nodes)?;
+        assert_eq!(sz, 6);
+        assert_eq!(asm.take(), hex!("6001602a6002"));
+
+        Ok(())
+    }
+
+    #[test]
+    fn assemble_include_asm() -> Result<(), Error> {
+        let f = new_file!(
+            r#"
+                jumpdest .a
+                pc
+                push1 .a
+                jump
+            "#
+        );
+        let nodes = nodes![
+            Op::Push1(Imm::from(1)),
+            Node::IncludeAsm(f.path().to_owned()),
+            Op::Push1(Imm::from(2)),
+        ];
+        let mut asm = Assembler::new();
+        let sz = asm.push_all(nodes)?;
+        assert_eq!(sz, 9);
+        assert_eq!(asm.take(), hex!("60015b586000566002"));
```

Comment:
Shouldn't this be `60015b586002566002`?

Reply:
I think the label should resolve to `0` because `include_asm` is including *assembled* ops, e.g. for initcode.